extensions/intellij/intellij_generator_plugin/src/test/resources/generator/RxTestBlocGoldenGenerator/RxTestBlocAlchemistGoldenGenerator_all.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter_test/flutter_test.dart';
+
+import '../../helpers/golden_helper.dart';
+import '../../helpers/models/scenario.dart';
+import '../factory/sample_factory.dart';
+import 'package:rx_bloc/rx_bloc.dart';
+import 'package:rx_bloc_list/models.dart';
```

Comment:
Same question as above for the rx_bloc_list and rx_bloc imports

Reply:
All unused imports were removed.